src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,187 @@
+        ),
+        icon: 'stop-hand',
+        sections: [
+          // Bitwarden free password manager
+          {
+            addonGuid: '{446900e4-71c2-419f-a6a7-df9c091e268b}',
+            header: i18n.gettext('Create and manage strong passwords'),
+            description: i18n.gettext(
+              'Password managers can help you create secure passwords, store your passwords (safely) in one place, and give you easy access to your login credentials wherever you are.',
+            ),
+            addonCustomText: i18n.gettext(
+              'Fully encrypted password protection. Store your data securely and access logins across devices.',
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/password-managers/',
+          },
+        ],
+      };
+    default:
+      return null;
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { errorHandler, i18n, match } = this.props;
+    const { slug } = match.params;
+    const content = getContent(slug, i18n);
+
+    if (!content) {
+      return null;
+    }
+
+    const guids = content.sections.map((section) => section.addonGuid);
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: errorHandler.id,
+      }),
+    );
+  }
+
+  getGuidesSections = (
+    sections: Array<SectionsType>,
+  ): React.ChildrenArray<React.Node> => {
+    const { clientApp, i18n, lang } = this.props;
+
+    return sections.map((section) => {
+      const exploreMore = i18n.sprintf(section.exploreMore, {
+        linkStart: `<a href="/${lang}/${clientApp}${section.exploreUrl}">`,
+        linkEnd: '</a>',
+      });
+
+      return (
+        <div className="Guides-section" key={section.exploreUrl}>
+          <h2 className="Guides-section-title">{section.header}</h2>
+          <p className="Guides-section-description">{section.description}</p>
+
+          {/* eslint-disable react/no-danger */}
+          <div
+            className="Guides-section-explore-more"
+            dangerouslySetInnerHTML={sanitizeHTML(exploreMore, ['a'])}
+          />
+          {/* eslint-enable react/no-danger */}
+        </div>
+      );
+    });
+  };
+
+  render() {
+    const { i18n, match } = this.props;
+    const { slug } = match.params;
+    const content = getContent(slug, i18n);
+
+    if (!content) {
+      return <NotFound />;
+    }
+
+    const { introText, icon, title, sections } = content;
+
+    return (
+      <div className="Guides-page">
+        <Helmet>
+          <title>{title}</title>
+        </Helmet>
```

Comment:
We will probably want to add the `HeadLinks` component (may need a rebase). All you have to do is adding `<HeadLinks />` below the `</Helmet>` block (not inside, outside, below).

Reply:
Write a test case for that.